### ISSUE TITLE
using conda, can't get it to work because of some python bug

### ISSUE THREAD

**Issue by martiliasf** [Illumina/GTCtoVCF#13]
```ImportError: /nas/longleaf/home/martyf/.local/lib/python2.7/site-packages/numpy/core/multiarray.so: undefined symbol: PyUnicodeUCS2_AsUTF8String```

my googling indicates the errors goes back to this

https://bugs.python.org/issue4434

**Comment by KelleyRyanM** [comment]
If you try to run a python script that contains only the line "import numpy", do you get the same error?

**Comment by martiliasf** [comment]
Somehow fixed the error by playing with my system environment.

Now I have a new fun error. First, I used the .bpm manifest that was provided. Then the convert said that it can't use that to process indels, so I switched to the .csv manifest .... then, it gives me this nice error

```2018-04-25 13:55:55,122 - GTC converter - WARNING - Provided manifest name: /pine/scr/m/a/martyf/GSA-24v1-0_A6.csv and manifest file used to generate GTC file: GSA-24v1-0_A2.bpm do not match, skipping````

So i try to recreate the gtc files using https://github.com/Illumina/BeadArrayFiles , which of course doesn't seem to like using csv as manifests.
